palette.txt

```diff
@@ -0,0 +1,27 @@
+background-color:3DFDCE3
+text: black
+tabs on hover, sections on hover: #F7B733
+h1, menu tabs: #FC4A1A
+nav bar: #4ABDAC
+
+green: #4ABDAC
+vermillion: #FC4A1A
+yellow: #F7B733
+grey: 3DFDCE3
+
+
+
+ice: #99D3DF
+water: #88BBD6
+plaster: #CDCDCD
+linen: #E9E9E9
+
+watermelon: #FF3B3F
+
+
+
+
+QUESTIONS:::::
+-what is up with the auto margin feature as a centering device?
+-i want for the three images on the index page to have the images disappear or appear on hover; also want these boxes to be clickable links, so can't click on a disappeared image
+-going crazy not being able to center the titles for these three boxes - anywhere would be cool, above, below, inside, just centered!
```

Comment:
I love that you have a file full of references to colors and design decisions just sitting around. That's such a good idea. For the future, I might suggest adding it to your gitignore, since this is basically your sandbox-of-ideas file, and you might not want that out in public. But I think it is very handy and now I know it's a thing, and I'm glad I saw it.